src/core/lib/security/authorization/cel_evaluation_engine.cc

```diff
@@ -16,31 +16,51 @@
 
 #include "src/core/lib/security/authorization/cel_evaluation_engine.h"
 
+std::unique_ptr<CelEvaluationEngine>
+CelEvaluationEngine::CreateCelEvaluationEngine(
+    const std::vector<envoy_config_rbac_v2_RBAC*>& rbac_policies) {
+  if (rbac_policies.size() == NumPolicies &&
```

Comment:
There are 3 valid inputs
- one deny policy followed by one allow policy
- just one deny policy
- just one allow policy

Reply:
Thanks for the review. Updated.